### ISSUE TITLE
Dropped Loot tooltips do not show item descriptions, special attacks or healing value

### ISSUE THREAD

**Issue by coolrox95** [MelvorIdle/melvoridle.github.io#618]
**Describe the bug**
Due to an error in the order of the code of the loadLoot() function, certain attributes of items that are normally displayed on hover in the bank are not.
The assignment of the tooltip HTML is done before the description, special attack or hp HTML is set, so they don't actually get displayed. Additionally the code that sets the special attack description doesn't work.

**To Reproduce**
1. Get loot that either has a description, special attack or healing value
2. Hover over the loot and note that the tooltip does not show these values

**Expected behavior**
Loot tooltips should have parity with the information displayed in bank tooltips.

**Screenshots**
Ancient 2H Sword displayed in the loot box
![image](https://user-images.githubusercontent.com/58208131/103851336-cb24b980-5066-11eb-858c-bd3565917cd2.png)
Displayed in the bank
![image](https://user-images.githubusercontent.com/58208131/103851371-e0014d00-5066-11eb-9868-b07e3dc7c32c.png)

**Browser**
Browser Agnostic Bug

**Fix**
Easiest way to fix this is to just move the bank tooltip HTML generation into a function
```js
function getItemTooltip(itemID) {
  let potionCharges = '', description = '', spec = '', hp = '', passive = '';
  const item = items[itemID];
  if (item.isPotion)
    potionCharges = //html
    `<small class='text-warning'>${item.potionCharges} Potion Charges</small><br>`;
  if (item.description !== undefined)
    description = //html
      `<small class='text-info'>${item.description}</small><br>`;
  if (item.hasSpecialAttack) {
    const specialAttack = playerSpecialAttacks[item.specialAttackID];
    spec = //html
      `<small class='text-success'>SPECIAL ATTACK<br><span class='text-danger'>${ 
        specialAttack.name 
      } (${ 
        specialAttack.chance 
      }%): </span><span class='text-warning'>${ 
        specialAttack.description 
      }</span></small> `;
    }
  if (item.canEat)
    hp = //html
      `<img class='skill-icon-xs ml-2' src='${ 
      CDNDIR 
      }assets/media/skills/hitpoints/hitpoints.svg'> <span class='text-success'>+${ 
      getFoodHealValue(itemID) 
      }</span>`;
  if (item.isPassiveItem !== undefined) {
    if (
      item.isPassiveItem &&
      dungeonCompleteCount[CONSTANTS.dungeon.Into_the_Mist] > 0
    )
      passive = //html
        `<br><small class="text-success">Passive Slot Compatible</small>`;
  }
  let itemTooltip = //html
    `<div class='text-center'><div class='media d-flex align-items-center push'><div class='mr-3'><img class='bank-img m-1' src='${ 
    getItemMedia(itemID) 
    }'></div><div class='media-body'><div class='font-w600'>${ 
    item.name 
    }</div>${ 
    potionCharges 
    }${description 
    }${spec 
    }<div class='font-size-sm'><img class='skill-icon-xs' src='${ 
    CDNDIR 
    }assets/media/main/coins.svg'> ${ 
    item.sellsFor 
    }${hp 
    }${passive 
    }<br></div></div></div></div>`;
  return itemTooltip;
}
```
And adjust loadLoot() as such:
```js
function loadLoot() {
  clearTimeout(loadLootTimer);
  loadLootTimer = setTimeout(function () {
    if (tooltipInstances.loot !== undefined) {
      tooltipInstances.loot.forEach((instance) => {
        instance.destroy();
      });
      tooltipInstances.loot.length = 0; // clear it
    } else tooltipInstances.loot = [];
    $('#combat-loot-container').html('');
    for (let i = 0; i < droppedLoot.length; i++) {
      let combatLoot = '';
      let tooltip = getItemTooltip(droppedLoot[i].itemID,'loot');
      combatLoot += `<div class="bank-item no-bg btn-light pointer-enabled" role="button" id="combat-loot-${i}" onClick="collectLoot(${i}, ${droppedLoot[i].itemID
        }, ${droppedLoot[i].qty});" style="margin: 10px;">
			<img class="bank-img p-3" src="${getItemMedia(
          droppedLoot[i].itemID
        )}"><br><div class="font-size-sm font-w600 text-center text-combat-smoke">${droppedLoot[i].qty
        }</div></div>`;
      $('#combat-loot-container').append(combatLoot);
      const instance = tippy('#combat-loot-' + i, {
        content: tooltip,
        placement: 'top',
        interactive: false,
        animation: false,
        allowHTML: true,
      });
      tooltipInstances.loot = tooltipInstances.loot.concat(instance);
    }
    $('#combat-loot-text').text(
      'Loot to Collect (' + droppedLoot.length + ' / ' + itemDropMax + ')'
    );
  }, 50);
}
```
And some simple code to test the tooltips
```js
droppedLoot.push({
  itemID: CONSTANTS.item.Shrimp,
  qty: 69
},
{
  itemID: CONSTANTS.item.Ancient_2H_Sword,
  qty: 69
},
{
  itemID: CONSTANTS.item.Amulet_of_Fishing,
  qty: 69
});
loadLoot();
```
![image](https://user-images.githubusercontent.com/58208131/103860356-b94c1200-5078-11eb-94b1-0a16cdc2c979.png)
![image](https://user-images.githubusercontent.com/58208131/103860371-bd782f80-5078-11eb-9747-5d32e85f756e.png)
![image](https://user-images.githubusercontent.com/58208131/103860385-c0732000-5078-11eb-8668-9c862178a330.png)

**Comment by Physalix** [comment]
Or just click the loot button, and look at what it does in the bank? This doesnt seem like a bug to me

**Comment by MelvorIdle** [comment]
Fixed in Alpha v0.21
